code/modules/power/supermatter/supermatter.dm

```diff
@@ -1107,7 +1107,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 	var/drops_core = FALSE
 	// Every 334 EER above POWER_PENALTY_THRESHOLD adds 10% chance for the anomalies to leave behind a core when neutralized
 	var/chance_bonus = clamp((10 * (power - POWER_PENALTY_THRESHOLD) / 334), 0, 95)
-	if(prob(5 + round(chance_bonus))) // 5% base chance to drop a core
+	if(prob(25 + round(chance_bonus))) // 25% base chance to drop a core
```

Comment:
Slap a fat fucking `- SKYRAT EDIT` or something on there and you're golden

Reply:
This already has edit comments